Sources/SwiftDocC/Indexing/RenderNode+Relationships.swift

```diff
@@ -108,12 +108,22 @@ public struct RenderRelationshipsGroup {
     
     /// Indicates if the group should nest sub-references or not. Default: false.
     public let referencesAreNested: Bool
-    
-    public init(name: String?, abstract: String?, references: [TopicRenderReference], referencesAreNested: BooleanLiteralType = false) {
+
+    /// Indicates if the group should be hidden from renderings. Default: false.
+    public let hidden: Bool
```

Comment:
This question also applies to `TaskGroupRenderSection`.

Reply:
If I understand what this is for, it sounds like this is behaving the same way as the [`@Links`](https://www.swift.org/documentation/docc/links) directive with the `list` visual style.

Reply:
The major purpose for adding the topic group (et al) that's hidden is so that the conceptual hierarchy of overload symbols isn't lost. For example, without adding the task group, the `linkable-entities.json` digest file shows all the overload symbols without any hierarchical relationship to the other symbols. This is why i didn't want to just reach for `excludeFromAutomaticCuration` - i still want tooling to be able to relate these nodes, and by the time we emit any structured information the only thing we use is `RenderNode.topicSectionVariants`.

What would you recommend for being able to relate these symbols without this task/topic group relationship?

Reply:
If it's only needed for the task groups in linkable-entities.json then I would suggest querying the topic graph for the information. Since the overload group is fully synthesized its topic sections are fully knowable from the topic graph.